data-es5.js

```diff
@@ -740,9 +740,8 @@ exports.tests = [
       ie9: true,
       firefox2: false,
       firefox5: true,
+      safari1: false,
       safari10_1: true,
-      safaritp: false,
```

Comment:
Def leave these; webkit, safari, and safaritp don't necessarily have the same features.

Reply:
Well, according to environments.json, they all belong in the same "family", so the results cascade.  But I am adding them back.

Reply:
Right but anything explicitly tested should never be removed - cascading is a sane default, but explicit results are never redundant.

Reply:
Well, as I stated in the first comment I haven't explicitly tested. :D